navigation/src/androidMain/kotlin/com/freeletics/khonshu/navigation/internal/StackSnapshot.kt

```diff
@@ -22,8 +22,11 @@ public class StackSnapshot internal constructor(
     internal val current: StackEntry<*>
         get() = entries.last()
 
+    internal val previous: StackEntry<*>
+        get() = entries.getOrNull(entries.size - 2) ?: startStackRootEntry
```

Comment:
If `current` is `startStackRootEntry` then what is returned here is technically wrong, but in practice `previous` is never used in that case.